content/Gems You Want To Have In Your Rails API - Juan Diego Rosales.md

```diff
@@ -0,0 +1,44 @@
+# Gems You Want To Have In Your Rails API
+
+*Juan Diego Rosales is a software developer on the Rootstrap team and writes about his experience working on different projects. You can also follow Juan on [LinkedIn](www.linkedin.com/in/juan-diego-rosales-02a05a144).*
```

Comment:
shouldn't this be in the `About` section, for example here
![image](https://user-images.githubusercontent.com/6373536/78814037-6556b480-79a4-11ea-92c9-dad4cc5d30fb.png)